tests/integration/targets/ndo_physical_interface/tasks/main.yml

```diff
@@ -0,0 +1,502 @@
+# Test code for the MSO modules
+# Copyright: (c) 2024, Anvitha Jain (@anvjain) <anvjain@cisco.com>
+
+# GNU General Public License v3[0]+ (see LICENSE or https://www.gnu.org/licenses/gpl-3[0].txt)
+
+- name: Test that we have an ACI MultiSite host, username and password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: mso_hostname, mso_username and mso_password.'
+  when: mso_hostname is not defined or mso_username is not defined or mso_password is not defined
+
+# CLEAN ENVIRONMENT
+- name: Set vars
+  ansible.builtin.set_fact:
+    mso_info: &mso_info
+      host: '{{ mso_hostname }}'
+      username: '{{ mso_username }}'
+      password: '{{ mso_password }}'
+      validate_certs: '{{ mso_validate_certs | default(false) }}'
+      use_ssl: '{{ mso_use_ssl | default(true) }}'
+      use_proxy: '{{ mso_use_proxy | default(true) }}'
+      output_level: '{{ mso_output_level | default("debug") }}'
+
+# QUERY VERSION
+- name: Query MSO version
+  cisco.mso.mso_version:
+    <<: *mso_info
+    state: query
+  register: version
+
+- name: Execute tasks only for MSO version > 4.3
+  when: version.current.version is version('4.3', '>=')
+  block:
+    - name: Remove fabric resource policy template
+      cisco.mso.ndo_template: &template_absent
+        <<: *mso_info
+        name: ansible_fabric_resource_policy_template
+        type: fabric_resource
+        state: absent
+
+    - name: Add fabric resource policy template
+      cisco.mso.ndo_template:
+        <<: *template_absent
+        state: present
+
+    # Create interface policy group in fabric policy template
+    - name: Remove fabric policy template
+      cisco.mso.ndo_template: &fabric_policy_template_absent
+        <<: *mso_info
+        name: ansible_fabric_policy_template
+        type: fabric_policy
+        state: absent
+
+    - name: Create a fabric policy template
+      cisco.mso.ndo_template:
+        <<: *template_absent
+        state: present
+    
+    # - name: Create interface policy group (physical policy)
```

Comment:
Are you planning to uncomment this when #540 is merged in?

Reply:
yes, rebased and uncommented